tmt/trying.py

```diff
@@ -0,0 +1,294 @@
+                self.debug("Use the default user plan config.")
+                return self.tree.plans(names=[f"^{plan_name}"], run=run)
+        except MetadataError:
+            self.debug("User config tree not found.")
+
+        # Use the default plan template otherwise
+        plan_name = DEFAULT_PLAN_NAME
+        plan_dict = tmt.utils.yaml_to_dict(DEFAULT_PLAN)
+        self.tree.tree.update(plan_dict)
+        self.debug("Use the default plan template.")
+        return self.tree.plans(names=[f"^{plan_name}"], run=run)
+
+    def check_plans(self, run: tmt.base.Run) -> None:
+        """ Check for plans to be used for testing """
+
+        # Search for matching plans if plan names provided
+        plan_names = list(self.opt("plan"))
+        if plan_names:
+            self.debug("Plan names filter", listed(plan_names, quote="'"))
+            self.plans = self.tree.plans(names=plan_names, run=run)
+            if not self.plans:
+                raise tmt.utils.GeneralError(
+                    f"No plan matching '{listed(plan_names)}' found.")
+
+        # Use default plans if no plan names requested
+        else:
+            self.plans = self.get_default_plans(run)
+
+        self.debug("Matching plans found\n" + tmt.utils.format_value(self.plans))
+
+        # Attach a login instace to each plan
+        for plan in self.plans:
+            plan.login = tmt.steps.Login(
+                logger=plan.provision._logger.descend(),
+                step=plan.provision,
+                order=tmt.steps.PHASE_END)
+
+    def welcome(self) -> None:
+        """ Welcome message with summary of what we're going to try """
+
+        parts = ["Let's try"]
+
+        # Test names
+        test_names = [click.style(test, fg="red") for test in self.tests]
+        if test_names and not self.opt("login"):
+            parts += [listed(test_names, 'test', max=3)]
+        else:
+            parts += [click.style("login", fg="red")]
+        parts += ["with"]
+
+        # Plan names
+        plan_names = [click.style(plan, fg="magenta") for plan in self.plans]
+        parts += [listed(plan_names, 'plan', max=3)]
+
+        # Image names
+        if self.image_and_how:
+            parts += ["on"]
+            image_names = [click.style(image, fg="blue") for image in self.image_and_how]
+            parts += [listed(image_names)]
+
+        self.info(" ".join(parts) + ".")
+
+    def choose_action(self) -> str:
+        """ Print menu, get next action """
+
+        def a(key: str) -> str:
+            """ Colorize the menu action """
+            return click.style(key, fg="bright_blue")
+
+        def c(key: str) -> str:
+            """ Colorize the menu key """
+            return click.style(key, fg="blue")
+
+        while True:
+            self.info(textwrap.dedent(f"""
+                What do we do next?
+
+                    {c('t')}{a('est')}       rediscover tests and execute them again
```

Comment:
How about more emphasis that first letter is the input? E.g. `[t]est` (t can stay blue). Or have it bold.. The color is not enough on my color schema (or for my eyesight:)

Reply:
Good idea! I didn't know that `click.style` can do `bold`. Looks better with 2a199a67?